circle.yml

```diff
@@ -22,6 +23,11 @@ test:
   # Grep error on the documentation
   override:
     - cat ~/log.txt && if grep -q "Traceback (most recent call last):" ~/log.txt; then false; else true; fi
+deployment:
+  push:
+    branch: master
+    commands:
+      - bash continuous_integration/push_doc.sh
```

Comment:
The the `general / branches / only / master` restriction is only there to prevent any PR to launch a circleCI build. I'll remove it in another PR to allow people to build the doc at will with noplot option.